src/middlewared/middlewared/plugins/alert.py

```diff
@@ -296,6 +298,18 @@ def restore(self, id):
                                     })
                                 except Exception:
                                     self.logger.error(f"Failed to create a support ticket", exc_info=True)
+                                    await self.middleware.call("mail.send", {
+                                        "cc": ["auto-support@ixsystems.com"],
```

Comment:
@william-gr which e-mail we should use here?

Reply:
Waiting on feedback from support people, will get back to you when I have an answer

Reply:
@themylogin please remove the cc line from the `except Exception` block. We will let the customer manually open a ticket in this scenario.